checks.d/hdfs_datanode.py

```diff
@@ -76,12 +76,10 @@ def _hdfs_datanode_metrics(self, jmx_uri):
 
         if beans:
 
+            # Only get the first bean
             bean = next(iter(beans))
             bean_name = bean.get('name')
```

Comment:
Could you `self.log.debug` the retrieved `bean_name` here, so that we can easily troubleshoot issues with the retrieved bean?